app/src/main/java/pm/gnosis/heimdall/ui/security/unlock/UnlockContract.kt

```diff
@@ -2,11 +2,14 @@ package pm.gnosis.heimdall.ui.security.unlock
 
 import android.arch.lifecycle.ViewModel
 import io.reactivex.Observable
+import io.reactivex.Single
 import pm.gnosis.heimdall.common.utils.Result
+import pm.gnosis.heimdall.security.FingerprintUnlockResult
 
 abstract class UnlockContract : ViewModel() {
     abstract fun checkState(): Observable<Result<State>>
     abstract fun unlock(password: String): Observable<Result<State>>
+    abstract fun watchFingerprint(): Observable<Result<FingerprintUnlockResult>>
```

Comment:
I know that I introduced that name ... but if we stick to our pattern it should be `observeFingerprint`